packages/clay-tabs/src/index.tsx

```diff
@@ -10,38 +10,35 @@ import Content from './Content';
 import Item from './Item';
 import TabPane from './TabPane';
 
+export type DisplayType = null | 'tabs' | 'underline';
+
 interface IProps extends React.HTMLAttributes<HTMLUListElement> {
 	/**
-	 * Justify the nav items according the tab content.
+	 * Determines how to is displayed.
 	 */
-	justified?: boolean;
+	displayType?: DisplayType;
 
 	/**
-	 * Applies a modern style to the tab.
+	 * Justify the nav items according the tab content.
 	 */
-	modern?: boolean;
```

Comment:
Sorry if this isn't very clear but unfortunately we still need to support this property to avoid breaking change, so we have to handle with this property still taking effect but using `displayType` overrides the behavior of `modern`.

We are going to remove this API in v4, so you can add this information in the document in the `BREAKING.md` package as a list, we are using this so we don't lose sight of the APIs we need to remove in the next major version.

Reply:
Ohh! Got it! At the first time, I thought that this was a change requested but considering as a bug. So in this case, I have to still support the `modern` behaviour with the new property `displayType`. However, `displayType` must override `modern` one. Yeah, I will add this information in `BREAKING.md`, to record it for the v4.

Reply:
Yes, exactly sorry for this confusion 😅!